src/makeRobot.js

```diff
@@ -38,7 +38,92 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    location: `${name}: x=0, y=0`,
+    goForward: (move) => {
+      if (move && move > 0) {
+        robot['coords']['y'] += move;
+
+        robot.location
+          = `${robot.name}: x=${robot.coords.x}, y=${robot.coords.y}`;
+      }
```

Comment:
```suggestion
      }
```
Это не надо если сделать location геттером.